columnencoder.h

```diff
@@ -23,11 +23,11 @@
 #include <map>
 #include <set>
 
-#ifdef BUILDING_JASP
-#include "jsonredirect.h"
-#else
-#include "lib_json/json.h"
-#endif
+// #ifdef BUILDING_JASP
```

Comment:
same here though, this can just be `#include  <json/json.h>` ? and no commentcrap

Reply:
Yes, noticed it as well but then lunch was ready! 🍲